environments/icds-staging/proxy.yml

```diff
@@ -1,4 +1,5 @@
 SITE_HOST: 'icds-staging.commcarehq.org'
+CAS_SITE_HOST: 'icds-staging.commcarehq.org'
```

Comment:
I don't think this is needed

Reply:
the below TASK was looking for this key : value

TASK [nginx : Drop site configuration] *****************************************************************************************************************************************************************************************************
fatal: [100.71.188.147]: FAILED! => {"msg": "'CAS_SITE_HOST' is undefined"}

Reply:
ah, I see - you need to drop the `cas_proxy` group from the proxy VM in `icds-staging/inventory/other.csv`:
```diff
  hostname,host_address,group 1,group 2,group 3,group 4,group 5,group 6,var: hostname,var: public_ip
- proxy0,100.71.188.147,proxy,cas_proxy,staticfiles,patch,control,formplayer,MUMGCCWCDSTGPX00,14.141.62.13
+ proxy0,100.71.188.147,proxy,,staticfiles,patch,control,formplayer,MUMGCCWCDSTGPX00,14.141.62.13
```